backend/src/connectors/vdb_file.py

```
@@ -122,85 +124,117 @@ class VDBFile:
         self.metadata = metadata
 
+    def download(self, path: str) -> Path: ...
     def get_text(self) -> str | list[str]: ...
 
 
 class GoogleDriveFile(VDBFile):
+    GOOGLE_EXPORT_MIME = {
+        "application/vnd.google-apps.document": "text/plain",
+        "application/vnd.google-apps.spreadsheet": "text/csv",
+        "application/vnd.google-apps.presentation": "text/plain",
+    }
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)